engines/lastexpress/debug.cpp

```diff
@@ -487,10 +487,7 @@ bool Debugger::cmdPlaySeq(int argc, const char **argv) {
 
 					// Handle right-click to interrupt sequence
 					Common::Event ev;
-					if (!_engine->getEventManager()->pollEvent(ev))
-						break;
-
-					if (ev.type == Common::EVENT_RBUTTONUP)
+					if (_engine->getEventManager()->pollEvent(ev) && ev.type == Common::EVENT_RBUTTONUP)
```

Comment:
This changes the behviour. The equivalent one would be ``(!_engine || ev.type)``. Unless you have a reason to change it.

Reply:
This is intentional. The old behaviour is "while the event queue is not empty and does not start with a right-click event, remove the first event and play the sequence", which does not make much sense (in practice, it just draws first few frames of the sequence and exits). The new behaviour is "play the sequence until right-click event (or till the end of the sequence)".